vilebot/src/main/java/com/oldterns/vilebot/handlers/user/QuotesAndFacts.java

```diff
@@ -61,7 +61,7 @@
     // update cache when new quotes/facts added
     private Map<String, Integer> dumpSize = new HashMap<>();
 
-    private static final String PASTEBIN_API_URL = Vilebot.getConfig().get("pastebinApiUrl");
+    private static final String PASTEBIN_API_URL = Vilebot.getConfig().get( "pastebinApiUrl" );
```

Comment:
+1 for fixing this formatting, never got around to doing it.

Reply:
`mvn` did it for me; if you run `mvn clean install` it will auto-format your code